modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -126,9 +126,10 @@ sudo service apache2 reload
 
 [source,console,subs="attributes+"]
 ----
-sudo -u {webserver-user} wget -P=/var/www https://download.owncloud.org/community/owncloud-10.2.0.tar.bz2 && \
-  sudo tar -xjf owncloud-10.2.0.tar.bz2 && \
-  sudo chown -R {webserver-user} {install-directory}
+cd /var/www/
+wget https://download.owncloud.org/community/owncloud-10.2.1.tar.bz2 && \
```

Comment:
This will still fail, permission denied.

Reply:
works on my machine. where are you testing?

Reply:
Than you are root and don't need the following sudo calls.

Reply:
wasn't my idea to include the sudo calls

Reply:
removed the sudo calls